### PR TITLE
Update PS Credits

### PR DESCRIPTION
**Pull request by bumbadadabum**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -170,6 +170,7 @@ Contributors
 - Ben Davies [Morfent] - Development
 - Ben Frengley [TalkTakesTime] - Development
 - Cody Thompson [Rising_Dusk] - Development
+- [HoeenHero] - Development
```
Comment (Zarel) [inline README.md]: He definitely goes in Major Contributors.

Have you asked him about crediting as real name?

Comment (Zarel) [reply README.md]: Also we should have jumbowhales in here.

Comment (bumbadadabum) [reply README.md]: I did ask and he didn't want his real name on there at this moment.

Comment (Zarel) [reply README.md]: also mathfreak231

File: README.md [hunk 2]
```diff
@@ -155,10 +155,11 @@ Retired Staff
 - Cathy J. Fitzpatrick [cathyjf] - Development, Sysadmin
 - Hugh Gordon [V4] - Research (game mechanics), Development
 - Juanma Serrano [Joim] - Development, Sysadmin
+- Leonardo Julca [Slayer95] - Development
 
 Major Contributors
 
-- Bär Halberkamp [bumbadadabum] - Development
+- [HoeenHero] - Development
```
Comment (Zarel) [inline README.md]: @HoeenCoder I say this to everyone: You should be credited by real name.

There are a few reasons I prefer real name:

1. It makes it easier for you to actually point to this in résumés and job interviews and say "that's me"
2. It makes the project look more professional
3. The entire point of crediting is to tell people who you are, not telling people who you are kind of defeats the purpose

**Comment by Zarel** [comment]
@whalemer @mathfreak231 Do you guys want to be credited? If so, we prefer real name for these reasons:

1. It makes it easier for you to actually point to this in résumés and job interviews and say "that's me"
2. It makes the project look more professional
3. The entire point of crediting is to tell people who you are, not telling people who you are kind of defeats the purpose

**Comment by Zarel** [comment]
Also @CheeseMuffin @MacChaeger

**Comment by Zarel** [comment]
Also anyone else I missed. Any change that isn't just a grammar correction is likely to be eligible to be credited.

File: README.md (hunk 2 above)
Comment (HoeenCoder) [reply README.md]: Alright, list me as Andrew Werner bumba

**Comment by sirDonovan** [comment]
@bumbadadabum can you also move me to staff please?

**Comment by CheeseMuffin** [comment]
Sure, name is Luke Harmon-Vellotti

**Comment by bumbadadabum** [comment]
Added whales and moo

**Comment by KrisXV** [comment]
@bumbadadabum can you remove my username?

**Comment by KrisXV** [comment]
i know but there is a pullreq open so i figured might as well utilize it :[

File: README.md [hunk 3]
```diff
@@ -155,16 +155,18 @@ Retired Staff
...
 
-- Bär Halberkamp [bumbadadabum] - Development
+- Andrew Werner [HoeenHero] - Development
 - Kevin Lau [Ascriptmaster] - Development, Art (battle animations)
 - Konrad Borowski [xfix] - Development
 - Quinton Lee [sirDonovan] - Development
 
 Contributors
 
+- Alexander B [mathfreak321] - Development
```
Comment (Zarel) [inline README.md]:
```suggestion
- Alexander B. [mathfreak321] - Development
```

Be consistent :p

**Comment by Zarel** [comment]
> `Zarel:bumbadadabum-patch-1`

wait, you put the branch on the official repo??? <_<

**Comment by bumbadadabum** [comment]
I thought I would try github online editing for this but uhh guess that was
a terrible idea it did this by default.

**Comment by Zarel** [comment]
Oh, I wasn't sure if this was done or not.

**Pull request merged**